airbyte-integrations/connectors/source-shopify/source_shopify/streams/base_streams.py

```diff
@@ -571,6 +570,9 @@ def read_records(self, stream_slice: Optional[Mapping[str, Any]] = None, **kwarg
 
 
 class IncrementalShopifyStreamWithDeletedEvents(IncrementalShopifyStream):
+
+    _stream_state: MutableMapping[str, Any] = {}
```

Comment:
This feels dangerous as a class property instead of an instance property. Is there a reason for that?

Reply:
Added the `initializer`, instead.